composer.json

```diff
@@ -4,8 +4,7 @@
     "keywords": ["Mockup"],
     "require": {
         "php": ">=5.4.1",
-        "symfony/symfony": "~2.0",
-        "claroline/kernel-bundle": "~6.0"
+        "claroline/distribution": "~7.0"
```

Comment:
Done, in https://github.com/hevinci/MockupBundle/commit/bdee62fea3804576ce1de42c24c3ecba960bde52. Closing, thanks.